nautobot_ssot/models.py

```diff
@@ -103,9 +103,8 @@ def get_absolute_url(self, api=False):
     def annotated_queryset(cls):
```

Comment:
This is used in 3 views `DataSourceTargetView`, `SyncListView` and `SyncView`. Can you confirm that this change doesn't negatively impact `DataSourceTargetView` and `SyncView`?